src/components/header-menu/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import HeaderMenuItem from '../header-menu-item';
+import { HeaderMenuProps } from './types';
```

Comment:
Типы лучше импортировать с `type`. Так их можно сразу отличить от импортов значений и при компиляции такие импорты будут сразу вырезаться

 ```suggestion
import type { HeaderMenuProps } from './types';
```